src/main/java/com/mtihc/regionselfservice/v2/plots/PlotControl.java

```diff
@@ -33,1336 +33,1179 @@
 import com.sk89q.worldguard.protection.regions.ProtectedRegion;
 import com.sk89q.worldguard.protection.regions.ProtectedRegion.CircularInheritanceException;
 
-public class PlotControl {
-
-	private PlotManager mgr;
 
-	public PlotControl(PlotManager manager) {
-		this.mgr = manager;
+public class PlotControl {
+    
+    private PlotManager mgr;
+    
+    public PlotControl(PlotManager manager) {
+	this.mgr = manager;
+    }
+    
+    public PlotManager getPlotManager() {
+	return this.mgr;
+    }
+    
+    public int getRegionCountOfPlayer(World world, String playerName) {
+	// get WorldGuard's region manager
+	RegionManager regionManager = this.mgr.getPlotWorld(world.getName()).getRegionManager();
+	
+	// get online player
+	Player p = Bukkit.getPlayerExact(playerName);
+	if (p != null) {
+	    // when player is online, use WorldGuard's method of counting regions
+	    return regionManager.getRegionCountOfPlayer(this.mgr.getWorldGuard().wrapPlayer(p));
 	}
 	
-	public PlotManager getPlotManager() {
-		return mgr;
+	// player is offline
+	
+	// get all regions
+	Collection<ProtectedRegion> regions = regionManager.getRegions().values();
+	if (regions == null || regions.isEmpty()) {
+	    return 0;
 	}
 	
-	public int getRegionCountOfPlayer(World world, String playerName) {
-		// get WorldGuard's region manager
-		RegionManager regionManager = mgr.getPlotWorld(world.getName()).getRegionManager();
-		
-		// get online player
-		Player p = Bukkit.getPlayerExact(playerName);
-		if(p != null) {
-			// when player is online, use WorldGuard's method of counting regions
-			return regionManager.getRegionCountOfPlayer(mgr.getWorldGuard().wrapPlayer(p)); 
-		}
-		
-		// player is offline
-		
-		// get all regions
-		Collection<ProtectedRegion> regions = regionManager.getRegions().values();
-		if(regions == null || regions.isEmpty()) {
-			return 0;
-		}
-		
-		// count owned regions
-		int count = 0;
-		for (ProtectedRegion region : regions) {
-			if(region.isOwner(playerName.toLowerCase())) {
-				count++;
-			}
+	// count owned regions
+	int count = 0;
+	for (ProtectedRegion region : regions) {
+	    if (region.isOwner(playerName.toLowerCase())) {
+		count++;
+	    }
+	}
+	return count;
+    }
+    
+    private static final HashSet<Byte> invisibleBlocks = new HashSet<Byte>();
+    
+    @SuppressWarnings("deprecation")
+    public static HashSet<Byte> getInvisibleBlocks() {
+	if (invisibleBlocks.isEmpty()) {
+	    invisibleBlocks.add((byte) Material.AIR.getId());
+	    invisibleBlocks.add((byte) Material.WATER.getId());
+	    invisibleBlocks.add((byte) Material.STATIONARY_WATER.getId());
+	    invisibleBlocks.add((byte) Material.LAVA.getId());
+	    invisibleBlocks.add((byte) Material.STATIONARY_LAVA.getId());
+	    invisibleBlocks.add((byte) Material.SNOW.getId());
+	    invisibleBlocks.add((byte) Material.LONG_GRASS.getId());
+	}
+	return invisibleBlocks;
+    }
+    
+    @SuppressWarnings("deprecation")

```

Comment:
```suggestion
```
I would suggest not to suppress "deprecation" warnings. I would either try to use not-deprecated code, or just keep letting it warn us.